### DISCUSSION TITLE
IconSax and Lit

### DISCUSSION THREAD

**Discussion by MM25Zamanian** [lit/lit#2953] [Show and tell]
After much effort to find the best way
I was able to make [this](https://github.com/MM25Zamanian/mmzmk/tree/main/packages/iconsax)
I will be very happy to help you

**Comment by kevinpschaaf** [comment]
Neat! 

I had to dig around a little bit to understand what you built, so to help others:

* Icon set based on [iconsax.io](https://iconsax.io)
* npm package is [@erbium/iconsax](https://www.npmjs.com/package/@erbium/iconsax)
* Playground demo: https://lit.dev/playground/#gist=28a9d848e7b25a1a838dee780800d126

<img width="890" alt="image" src="https://user-images.githubusercontent.com/1354932/171968855-2ebde0ef-ea7c-4702-9fee-4709e1a381db.png">

**Reply by ynxtynxt** [reply]
This look great. Btw there is a typo backslash at the end of the playground link.

**Reply by kevinpschaaf** [reply]
Thanks, fixed.
